tests/integration-all/event-bridge/service/serverless.yml

```diff
@@ -13,6 +13,14 @@ functions:
           pattern:
             source:
               - serverless.test
+  eventBusDefaultArn:
```

Comment:
Great move, on covering that with integration tests!

Reply:
I tested the integration tests on my AWS account and it created everything just fine. I have also run the deployment on our app with my modified code and that worked as well.